catalog/app/containers/Bucket/PackageDialog/PackageCreationForm.tsx

```diff
@@ -625,6 +624,8 @@ function PackageCreationForm({
                       validate={validateFiles as FF.FieldValidator<$TSFixMe>}
                       validateFields={['files']}
                       errors={{
+                        conflict:
+                          'There are files with identical names. Rename or remove them please',
```

Comment:
```suggestion
                          'There are entries with conflicting logical keys. Rename or remove them please',
```
not sure if this is more user-friendly for less-tech-savvy people, but it seems more correct